ledger_device_sdk/src/nbgl.rs

```diff
@@ -20,6 +22,11 @@ pub struct Field<'a> {
     pub value: &'a str,
 }
 
+pub struct CField {
```

Comment:
This doesn't need to be a public struct it seems we will only use it internally.